frontend/src/types.ts

```diff
@@ -523,14 +523,14 @@ export type TagContent = {
 
 export type ImageTagInfo = {
   name: string;
-  content: TagContent;
+  content?: TagContent;
   recommended: boolean;
   default: boolean;
 };
 
 export type ImageInfo = {
   name: string;
-  tags: ImageTagInfo[];
+  tags?: ImageTagInfo[];
```

Comment:
According to the backend, this is always an array.
```suggestion
  tags: ImageTagInfo[];
```